ocb/random_vectors_generate_script.go

```diff
@@ -0,0 +1,87 @@
+package ocb
+
+import (
+	"crypto/aes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	mathrand "math/rand"
+	"os"
+	"testing"
+	"time"
+)
+
+// The following scripts create random test vectors for the ocb package
+// and writes them to a file.
+// In the test vectors provided by RFC 7253, the "bottom"
+// internal variable, which defines "offset" for the first time, does not
+// exceed 15. However, it can attain values up to 63.
+
+// These vectors include key length in {128, 192, 256}, tag size 128, and
+// random nonce, header, and plaintext lengths.
```

Comment:
These comments have valuable information though, you could copy them to random_vectors.go.

Reply:
+1 (actually those comments belong to the ocb package).